client/src/pages/people/Form.js

```diff
@@ -683,7 +684,8 @@ const PersonForm = ({
                   name="endOfTourDate"
                   component={FieldHelper.SpecialField}
                   value={values.endOfTourDate}
-                  onChange={value => setFieldValue("endOfTourDate", value)}
+                  onChange={value =>
+                    setFieldValue("endOfTourDate", moment(value).endOf("day"))}
```

Comment:
React gives a warning:
```
Warning: Failed prop type: Invalid prop `value` supplied to `CustomDateInput`, expected one of type [string, number].
CustomDateInput@http://localhost:3000/static/js/anet.js:43723:12
SpecialField@http://localhost:3000/static/js/anet.js:48486:15
FastFieldInner@http://localhost:3000/static/js/anet.js:304721:30
FormikConnect(FastFieldInner)
DictionaryField@http://localhost:3000/static/js/anet.js:45793:26
[…]
```
Changing to `setFieldValue("endOfTourDate", moment(value).endOf("day").toDate())` fixes this.